pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -364,6 +364,14 @@ func TestEnoughRequests(t *testing.T) {
 			wantStatus:                framework.NewStatus(framework.Unschedulable, getErrReason(v1.ResourceMemory)),
 			wantInsufficientResources: []InsufficientResource{{v1.ResourceMemory, getErrReason(v1.ResourceMemory), 16, 5, 20}},
 		},
+		{
+			pod: newResourcePod(
+				framework.Resource{MilliCPU: 1, Memory: 1, ScalarResources: map[v1.ResourceName]int64{extendedResourceB: 1}}),
```

Comment:
I know this isn't like that in the existing test, but a test actually reads better if the values are inlined.
Also, add a resource that shouldn't be ignored.